### PR TITLE
Add tinydb v3.2.0

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

**Comment by ericdill** [comment]
@msiemens I am adding tinydb to conda-forge for linux, mac and windows. This will allow users to easily install this with [conda] (http://conda.pydata.org/docs/) (conda install -c conda-forge tinydb). The maintenance burden here is light as all testing and releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe which will be located at https://github.com/conda-forge/tinydb-feedstock shortly after this PR is finished and merged. 

If you would like to be added as a maintainer (to make new releases for instance), I can happily add you to the maintainers list. If not, that is totally fine too!

**Comment by msiemens** [comment]
Thanks @ericdill for taking up this cause. There's an issue at msiemens/tinydb#71 about having TinyDB on Anaconda. This PR would close it, right?

> If you would like to be added as a maintainer (to make new releases for instance), I can happily add you to the maintainers list. If not, that is totally fine too!

I'd glad to be added as maintainer. That should make new versions available faster (as long as I don't forget to include it in my version release workflow).

**Comment by ericdill** [comment]
> This PR would close it, right?

Sure would.  This PR makes `tinydb` available from the `conda-forge` channel on anaconda.org, so your users could install with `conda install tinydb -c conda-forge`

> I'd glad to be added as maintainer. 

Ok I'll add you. Thanks for the rapid response!

**Comment by ericdill** [comment]
@ocefpaf could you give this a once-over? Pretty simple package as it is pure python with few deps

File: recipes/tinydb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        TinyDB is a tiny, document oriented database optimized for your 
+        happiness :)
+
+extra:
+    recipe-maintainers:
+        - ericdill
+        - licode
+        - msiemens
+        - tacaswell
+
```
Comment (jakirkham) [inline recipes/tinydb/meta.yaml]: nit: please remove extra newline.

File: recipes/tinydb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    summary: >
+        TinyDB is a tiny, document oriented database optimized for your 
+        happiness :)
+
+extra:
+    recipe-maintainers:
+        - ericdill
+        - licode
+        - msiemens
+        - tacaswell
```
Comment (jakirkham) [inline recipes/tinydb/meta.yaml]: @licode @tacaswell, are you guys ok being added as maintainers here?

**Comment by jakirkham** [comment]
Basically LGTM, @ericdill. Minor nit and wanting to confirm with other maintainers that they are ok being added here.

**Comment by ericdill** [comment]
Thanks. I just lost wifi at the airport, so will fix the whitespace issue when I land.  @licode and @tacaswell are dayjob-coworkers and have agreed to help me maintain conda recipes for our Python stack.

**Comment by jakirkham** [comment]
No worries. It's a minor thing.

Maybe if they want to confirm that they are generally ok with being listed on this that you add @ericdill, we can just note that in a comment in this issue ( https://github.com/conda-forge/staged-recipes/issues/139 ) and save everyone a few questions/pings.

**Comment by ericdill** [comment]
@jakirkham Removed the newline. licode and tacaswell have given me their permission to add them to recipes over in #139.  Thanks for suggesting that they post there.

**Comment by ocefpaf** [comment]
LGTM.Thanks @ericdill!

**Comment by ocefpaf** [comment]
Missed the `[skip ci] [skip appveyor]` in your last commit and I was trying to get the CIs to run again :smile:

**Comment by ericdill** [comment]
Thanks @ocefpaf !

**Pull request merged**